src/drinx/transform.py

```
@@ -10,24 +10,57 @@
+        aux = (
+            tuple(getattr(obj, f) for f in static_init_fields),
+            tuple(getattr(obj, f) for f in private_fields_names),
+        )
         return keyed_leaves, aux
 
     def unflatten(aux, leaves):
-        kwargs = {**dict(zip(static_fields, aux)), **dict(zip(dynamic_fields, leaves))}
-        return cls_(**kwargs)
+        static_init_values, private_values = aux
+        init_kwargs = {
+            **dict(zip(static_init_fields, static_init_values)),
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)